### DISCUSSION TITLE
Any success with Ubuntu?

### DISCUSSION THREAD

**Discussion by Bradyns** [audiamus/BookLibConnect#101] [Q&A]
### G'day!

I'm currently across the continent from my windows machine and am using my laptop with Kubuntu.

Before undertaking the (most-times painful) task of running this through wine/bottles, I was wondering if anyone has had success in the past, and whether they ran into hiccups with both BookLibConnect and the AAC converter.

On the face of it, it looks fairly no-frills as far as dependencies goes, but I know it's never that straight forward.

I've used the programs many times to much success, so thought I'd reach out.

_Regards,
Bradyns_

**Comment by typingbeaver** [comment]
I get to run it with Wine using GNOME Bottles, but am running into errors. :/

![Screenshot_20231104_115008](https://github.com/audiamus/BookLibConnect/assets/74824542/f3c249c0-c321-4c6d-90a0-0f121cabc35c)

![Screenshot_20231104_115137](https://github.com/audiamus/BookLibConnect/assets/74824542/0579e771-02c4-4d5a-b721-7af111c61d50)

**Reply by XXBrain** [reply]
Hi, I'm on Fedora and it's running without errors now. Iuse winetrick, a in 64bit wine environment.

**Comment by colvdv** [comment]
Check this out: https://github.com/audiamus/BookLibConnect/discussions/196
